src/lib/utils/simulator.ts

```diff
@@ -19,6 +19,31 @@ export class TxSimulator {
     }
   }
 
+  getHeaders() {
+    return {
+      'Content-Type': 'application/json',
+      'X-Access-Key': this.accessKey,
+    }
+  }
+
+  /**
+   * Gives public access to passed simulation and outputs public url to console.
+   * @param simulationId ID of the simulation to be made public.
+   */
+  async share(simulationId: string) {
+    const requestOptions = {
+      method: 'GET',
+      headers: {
+        'X-Access-Key': this.accessKey,
+      },
+    }
+    const url = `https://api.tenderly.co/api/v1/account/${this.user}/project/${this.project}/simulations/${simulationId}/share`
+    const res = await fetch(url, requestOptions)
```

Comment:
This always seems to fail, not sure why.

Reply:
If I call this url with Postman for example it will work and make the simulation public.

So I wonder if it's some kind of async issue. Tried with `sleep()` tho and didn't seem to make a difference.